### PR TITLE
p5:add page

### PR DESCRIPTION
**Pull request by hill**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by hill** [comment]
All good @sbrl - thanks for that!

File: pages/common/p5.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# p5-manager
```
Comment (agnivade) [inline pages/common/p5.md]: Shouldn't this be `p5` ? Since the command name is p5.

Comment (hill) [reply pages/common/p5.md]: Yeah wasn't sure what was correct to use here - the actual package is called p5-manager (as p5 is a seperate javascript library). Should this be changed to the command name or package name?

Comment (sbrl) [reply pages/common/p5.md]: I think the idea is that it's the command name - which should be the same as the filename IIRC.

Comment (hill) [reply pages/common/p5.md]: Ok, most recent commit has name changed 😁

**Pull request merged**